### ISSUE TITLE
Link to blog post

### ISSUE THREAD

**Issue by knut-hildebrandt** [akirk/enable-mastodon-apps#8]
In the account that follows the one hosted on WP in all the toots there appears a link to the blog post representing these toots. Since what I see there is exactly the contents of the toot I consider this link as not necessary. If it is there for some reason I do not know I'd wish for an option in settings to suppress it.

Please consider this as a suggestion or wish not an essential bug.

**Comment by mikewmerritt** [comment]
I think this is related to the ActivityPub plugin, not this one. I've experimented with removing the link since it seemed spammy for just a social networking status, but came to find out it's necessary for webmention, Brid.gy and other purposes.

A setting in the AP plugin might be nice, though.

**Comment by akirk** [comment]
Indeed, I personally have set the ActivityPub content to custom and just `[content]` but I'll coordinate with @pfefferle to see what a good solution could be.
